apps/meteor/app/api/server/v1/users.ts

```diff
@@ -500,8 +500,12 @@ API.v1.addRoute(
 				return API.v1.failure('Username is already in use');
```

Comment:
Just need help getting the password reset

Reply:
I have had an email stolen which I hadn't lots of my business portfolio photos are link to the email so I'm trying to get it back For personal reasons

Reply:
I'm sorry to read that, but this is not the place to ask for this kind of guidance.

 could you inform what is the server url? its hosted by rocket.chat?